package.json

```diff
@@ -23,11 +23,13 @@
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "start": "ember serve",
     "test": "ember test",
-    "test:types": "ember try:each --config-path=ember-try-typescript.js",
+    "pretest": "npm-run-all prepublish:*",
+    "posttest": "npm run clean",
+    "test:types": "ember try:each --config-path=./config/ember-try-typescript.js",
```

Comment:
I *believe* this config should work correctly in terms of generating the types we need. It will run `downlevel-dts` *prior* to running the `ember-try` suite for tests—both when running `test:types` and when running the full `ember try:each` suite in `test:all`—so users can verify directly *and* it will be correctly configured on CI runs, so that we actually check the type tests against the publicly generated types.

However, it doesn't currently actually _invoke_ the type tests at `types/modifier-test.ts`—they're not emitted, so they won't be checked!—so we need to change *that* configuration in some way. 🤔 

@dfreeman @jamescdavis do you have thoughts here?

Reply:
The only things I've come up with are to *manually* copy the tests into the target directory from down-leveling, or to put the type tests in a spot where they get included automatically by `ember ts:precompile` (probably suggesting a need to change the behavior of `ember-cli-typescript` to support it).

Reply:
I think I came up with something reasonable; see 21c57a2